src/parameters/path/user_id.yaml

```
@@ -0,0 +1,6 @@
+name: user_id
+in: path
+required: true
+description: The id of the signed-in user
```

```suggestion
description: The id of a user
```
The user specified might not be the one doing the request.